### PR TITLE
Book entity is created, connection to DB is created

### PR DESCRIPTION
**Pull request by Alexsandr-Yablunovskyi**

### CODE REVIEW

**Review by Cayman4010 — APPROVED** [review]
Good job!

File: src/main/java/yablunovskyi/bookstore/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Table(name = "books")
+@Data
```
Comment (liliia-ponomarenko) [inline src/main/java/yablunovskyi/bookstore/model/Book.java:14]: Lombok is a great library but be [attentive when using it with Entity](https://jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/) JFYI

File: src/main/java/yablunovskyi/bookstore/model/Book.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    
+    @Column(nullable = false, unique = true)
+    private String isbn;
+    
+    @Column(nullable = false)
+    private BigDecimal price;
+    
+    private String description;
+    
+    private String coverImage;
```
Comment (liliia-ponomarenko) [inline src/main/java/yablunovskyi/bookstore/model/Book.java:34]:
```suggestion
    private BigDecimal price;    
    private String description;
    private String coverImage;
```

**Review by liliia-ponomarenko — APPROVED** [review]
Great! Fix my comments before merging ;)

**Pull request merged**